pisek/jobs/jobs.py

```diff
@@ -184,6 +184,10 @@ def _access_file(self, filename: TaskPath) -> None:
         """Add file this job depends on."""
         self._accessed_files.add(filename.path)
 
+    @property
+    def accessed_files(self) -> MutableSet[str]:
+        return self._accessed_files
```

Comment:
What's the point of making this a property instead of a public field?
We're returning a `MutableSet` anyway, so it seems it's allowed to modify it from the outside.